ui-frontend/packages/catalog-ui-search/src/main/webapp/component/fields/date-range.tsx

```diff
@@ -53,6 +53,7 @@ export const DateRangeField = ({
   return (
     <DateRangeInput
       allowSingleDayRange
+      minDate={new Date('Jan 1, 1900')}
```

Comment:
Feels like these would be worth factoring out as exported variables in date helpers.

Reply:
Agreed, thought that like a few minutes after I put the PR up.